public/locales/en.json

```diff
@@ -189,8 +189,8 @@
                 "description": "Sorry, only the owner can edit this note."
             },
             "limitReached": {
-                "title": "Reach the limit",
-                "description": "Sorry, you've reached the maximum length this note can be.",
+                "title": "Reached the limit",
+                "description": "Sorry, you've reached the maximum length this note can be ({{maxLength}} characters).",
```

Comment:
```suggestion
                "description": "Sorry, you've reached the maximum length this note can only be ({{maxLength}} characters).",
```

Reply:
I don't think that adding this filler word is really required.

Reply:
"Sorry, you've reached the maximum length for notes. Notes can be up to {{maxLength}} characters long."

Yes that's even longer, but doesn't sound as terse.